domains/achintya.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "Moon4546X",
+           "email": "",
+           "discord": "713352608172998707"
+        },
+    
+        "record": {
+            "TXT": "918ba5486b5908dce20187856dbefd"
```

Comment:
TXT file alone doesn't do anything.

Reply:
this is my first time doing this, I am having trouble what should I add next?

Reply:
Do you have a website at least?

Reply:
yes I do

Reply:
Then you can check https://is-a.dev/docs to see what records we support and what to do.